src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs

```diff
@@ -752,7 +752,7 @@ private PreLoginHandshakeStatus ConsumePreLoginHandshake(bool encrypt, bool trus
 
             if (!_physicalStateObj.TryProcessHeader()) { throw SQL.SynchronousCallMayNotPend(); }
 
-            if(_physicalStateObj._inBytesPacket > TdsEnums.MAX_PACKET_SIZE || _physicalStateObj._inBytesPacket < 0)
+            if (_physicalStateObj._inBytesPacket > TdsEnums.MAX_PACKET_SIZE || _physicalStateObj._inBytesPacket < 0)
```

Comment:
This is a result of Visual Studio's auto code alignment. Earlier code was out of indent, and VS fixed.
I don't see any reason to go back to wrong indentation just because it is not related with my feature.